assets/airports/mdsd.json

```diff
@@ -0,0 +1,4005 @@
+                "35": [["SD120", "A70|S220-"], ["LETIR", "A50|S210-"], ["OTVOG", "A30|S210-"]]
+            },
+            "draw": [["KODIX", "SD111", "KEPTI"], ["KODIX", "SD120", "LETIR"]]
+        },
+        "OKOSO": {
+            "icao": "OKOSO",
+            "name": "Ohkohsoh",
+            "suffix": {"17":"1G", "35":"1H"},
+            "entryPoints": {
+                "ETBOD": ["ETBOD"]
+            },
+            "body": [["OKOSO", "A160|S250-"]],
+            "rwy": {
+                "17": [["SD110", "A80|S220-"], ["KEPTI", "A30|S210-"]],
+                "35": [["SD127", "A100|S220-"], ["URLOR", "A40|S210-"], ["VINRO", "A30|S210-"]]
+            },
+            "draw": [["OKOSO", "SD110", "KEPTI"], ["OKOSO", "SD127", "URLOR", "VINRO"]]
+        },
+        "RIBES": {
+            "icao": "RIBES",
+            "name": "Reebaes",
+            "suffix": {"17":"1G", "35":"1H"},
+            "entryPoints": {
+                "PTA": ["PTA", ["BORAS", "A190|S270"]]
+            },
+            "body": [["RIBES", "A140|S250-"]],
+            "rwy": {
+                "17": [["SD112", "A70|S220-"], ["KEPTI", "A30|S210-"]],
+                "35": [["SD121", "A70|S220-"], ["LETIR", "A50|S210-"], ["OTVOG", "A30|S210-"]]
+            },
+            "draw": [["RIBES", "SD112", "KEPTI"], ["RIBES", "SD121", "LETIR"]]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "MDSD",
+            "destination": "",
+            "category": "departure",
+            "route": "MDSD.AVRES.KARUM",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 2,
+            "airlines": [
+                ["ava",      2],
+                ["cmp",      4],
+                ["fdx",      2],
+                ["n/fastGA", 3],
+                ["pwd",      4],
+                ["ups",      2]
+            ]
+        },
+        {
+            "origin": "MDSD",
+            "destination": "",
+            "category": "departure",
+            "route": "MDSD.KANAM.DCR",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 2,
+            "airlines": [
+                ["amx",      1],
+                ["ava",      2],
+                ["cmp",      4],
+                ["fdx",      2],
+                ["n/fastGA", 3],
+                ["pwd",      4],
+                ["ups",      2]
+            ]
+        },
+        {
+            "origin": "MDSD",
+            "destination": "",
+            "category": "departure",
+            "route": "MDSD.KERSO.ASIVO",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 4,
+            "airlines": [
+                ["aal",      4],
+                ["aea/long", 2],
+                ["afr/long", 2],
+                ["cfg",      2],
+                ["cmp",      4],
+                ["dal",      4],
+     
```

Comment:
I actually couldn't find the restricted areas in the sector file, but I could add them to the video map myself, it shouldn't be too hard since I already have the coordinates.

That area is kinda weird and I discussed it with @pedriglieri in #288. Basically, aircraft departing MDSD must contact the MDPP TMA when leaving the sector shown in SkyVector, but arriving aircraft should assume that the MDSD TMA has full control over that area. We agreed to include that area to the TMA because it gives the player more time to control the arrivals and it also just looks better.